dotnet/src/SemanticKernel/Planning/PlannerConfigBase.cs

```diff
@@ -18,4 +24,51 @@ public abstract class PlannerConfigBase
     /// A list of functions to exclude from the plan creation request.
     /// </summary>
     public HashSet<string> ExcludedFunctions { get; } = new();
+
+    /// <summary>
+    /// When using <see cref="Memory"/> to get relevant functions,
+    /// this list of functions will be included regardless of relevancy.
+    /// </summary>
+    public HashSet<string> IncludedFunctions { get; } = new();
```

Comment:
what is the string here? Just Function name? Should we make this 
`HashSet<(string,string)>` for (skillName, functionName)?

Reply:
Yes just function name. I think we could do that, though, yes.